server.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mysql = require('mysql');
+
+const app = express();
+app.use(bodyParser.json());
+
+const db = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: '',
```

Comment:
Avoid storing database credentials directly in your code. Consider using environment variables for better security.